### PR TITLE
add conda recipe and rever

### PR DESCRIPTION
**Pull request by dragonyanglong**

### CODE REVIEW

**Comment by dragonyanglong** [comment]
Hi @sbillinge , shall we converge this PR?

File: requirements/run.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 python >=3.6
-matplotlib
+matplotlib
```
Comment (sbillinge) [inline requirements/run.txt:2]: we might just need matplotlib-base here

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{% set reponame = "mpl-stylesheets" %}
```
Comment (sbillinge) [inline recipe/meta.yaml:1]: please check if it is usual to put the conda recipe in the repo of the actual package.  I am not sure.

File: rever.xsh [hunk 3]
```diff
@@ -1,11 +1,11 @@
-$PROJECT = 'pkgname'
+$PROJECT = 'billinge_style'
```
Comment (sbillinge) [inline rever.xsh:1]: I think this is not right now, right?

**Review by sbillinge — COMMENTED** [review]
few more comments.  Sorry it is so nit picky, but when we get this all set up it makes subsequent releases very easy!

File: recipe/meta.yaml (hunk 2 above)
Comment (dragonyanglong) [reply recipe/meta.yaml:1]: Oh I see. You are right. conda recipe would be in a fork of conda-forge repo. Thanks Simon!

**Comment by dragonyanglong** [comment]
Thanks Simon for the comments. i have updated and pushed. I also rebased. Please review.

File: bg_mpl_stylesheet/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+__version__ = '0.1.0'
```
Comment (sbillinge) [inline bg_mpl_stylesheet/__init__.py:1]: needs a line-end

**Review by sbillinge — COMMENTED** [review]
thanks, I don't see anything wrong now, so I am ready to merge.  Just one line-break needed since GitHub is complaining about it.   Maybe it would be good to run it through black or flake8 for style too to catch any more things like that.

**Comment by dragonyanglong** [comment]
Sure, I just ran flake8, and updated the coding style accordingly. pushed.

**Pull request merged**